vmdk_plugin/E2E_Tests/e2e_verifyVolmProperties_test.go

```diff
@@ -0,0 +1,133 @@
+		fmt.Println("running end_to_end tests - current time: ", time.Now())
+		vmIp = vms[vmIndx]
+
+		fmt.Println("Running test on VM - ", vmIp)
+		dockerCliCheck = VolmPropsUtil.IsDockerCliCheckNeeded(vms[vmIndx])
+		for i := 0; i < len(volSizes); i++ {
+			for k := 0; k < len(formatTypes); k++ {
+				fmt.Println("Creating a volume of Format Type - ", formatTypes[k])
+				volName := TestInputParamsUtil.GetVolumeNameWithTimeStamp("dockerVol")
+				_, err := TestUtil.InvokeCommand(vms[vmIndx], "docker volume create --driver=vmdk --name="+volName+" -o size="+volSizes[i]+" -o diskformat="+formatTypes[k])
+				if err != nil {
+					fmt.Println("Failed to create a volume named: ", volName)
+					log.Fatal(err)
+				} else {
+					fmt.Println("\n successfully created a  volume -- ", volName)
+					dockerVolmRmvCmd = dockerVolmRmvCmd + " " + volName
+				}
+				volmPropertiesAdminCli := VolmPropsUtil.GetVolumePropertiesAdminCli(volName, esx)
+				expctdPropsAdmin := []string{"100.00MB", formatTypes[k], "detached"}
+				if !contains(expctdPropsAdmin, volmPropertiesAdminCli) {
+					log.Fatal("Volm properties on admin cli do not matches the expected values")
+				}
+
+				if dockerCliCheck {
+					volmPropertiesDkrCli := VolmPropsUtil.GetVolumePropertiesDockerCli(volName, vms[vmIndx])
+					expctdPropsDkr := []string{"100MB", formatTypes[k], "<no value>"}
+					if !contains(expctdPropsDkr, volmPropertiesDkrCli) {
+						log.Fatal("Volume properties on admin cli do not matches the expected values")
+					}
+				}
+				TestUtil.InvokeCommand(vms[vmIndx], "docker run -d -P --volume-driver=vmdk -v "+volName+":/vol --name e2e busybox tail -f /dev/null")
+				// Verifying attached to the vm field
+				vmNameFrmAdminCli := VolmPropsUtil.GetVmAttachedToVolUsingAdminCli(volName, esx)
+				if dockerCliCheck {
+					vmNameFrmDockerCli := VolmPropsUtil.GetVmAttachedToVolUsingDockerCli(volName, vms[vmIndx])
+					if vmNameFrmDockerCli != vmNameFrmAdminCli {
+						log.Fatalf("Attached-to-VM field for volume from docker cli is [%s]. And attched-to-vm field from admin cli is [%s]", vmNameFrmDockerCli, vmNameFrmAdminCli)
+					}
+					volmPropertiesDkrCli := VolmPropsUtil.GetVolumePropertiesDockerCli(volName, vms[vmIndx])
+					expctdPropsDkr := []string{"100MB", formatTypes[k]}
+					if !contains(expctdPropsDkr, volmPropertiesDkrCli) {
+						log.Fatal("Volume properties on admin cli do not matches the expected values")
+					}
+				}
+				volmPropertiesAdminCli = VolmPropsUtil.GetVolumePropertiesAdminCli(volName, esx)
+				expctdPropsAdmin = []string{"100.00MB", formatTypes[k]}
+				if !contains(expctdPropsAdmin, volmPropertiesAdminCli) {
+					log.Fatal("Volume properties on admin cli do not matches the expected values")
+				}
+				VolmPropsUtil.ExecCmd(vms[vmIndx], "docker stop e2e ; docker rm e2e")
+			}
+		}
+	}
+}
+
+// method checks if the output from the ssh commands contains the expected values
+func contains(volmProps []string, op string) bool {
```

Comment:
need to improve method name .. it is misleading and lead to confusion with strings.contain() method